### PR TITLE
📝(docs) add useful commands to "contributing" in README

### PR DESCRIPTION
**Pull request by Leobouloc**
## Purpose

As a new developper on Ralph, it is not obvious what commands are useful and where to find them. This commit (inspired by the [doc of OBC](https://github.com/openfun/open-badges-client#hack-on-the-project)) adds info on testing, linting, and updating dependancies to README in the contributing section.

## Proposal

- [x] update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,39 @@ We try to raise our code quality standards and expect contributors to follow
 the recommendations from our
 [handbook](https://handbook.openfun.fr).
 
+### Usefull commands
```
Comment (Leobouloc) [inline README.md:174]:
```suggestion
### Useful commands
```

File: README.md [hunk 2]
```diff
@@ -171,6 +171,39 @@ We try to raise our code quality standards and expect contributors to follow
...
+
+```
+$ make lint
+```
+
+If you had new dependencies to the project, you will have to rebuild the Docker
+image (and the development environment):
+
+```
+$ make down && make build && make dev
```
[the remark this answers is not in the document]
Comment (quitterie-lcs) [reply README.md:198]: IMO, `make down` is too strong for this use case. 
Furthermore, using this combination of commands after a `make down` omits the `es-index` in the `make bootstrap`, which is necessary if your remove the containers from your dev stack.

[the remark this answers is not in the document]
Comment (wilbrdt) [reply README.md:198]:
```suggestion
$ make down && make bootstrap
```
`make down` seems to be necessary, right?

File: README.md [hunk 3]
```diff
@@ -171,6 +171,39 @@ We try to raise our code quality standards and expect contributors to follow
...
+```
+
+You can explore all other target using:
```
Comment (wilbrdt) [inline README.md:201]:
```suggestion
You can explore all other rules using:
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (quitterie-lcs) [reply README.md:198]: @wilbrdt Yes, you're right

File: README.md [hunk 4]
```diff
@@ -171,6 +171,39 @@ We try to raise our code quality standards and expect contributors to follow
...
```
Comment (quitterie-lcs) [inline README.md:201]:
```suggestion
You can explore all available rules using:
```

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**